components/input-group/src/Right.js

```diff
@@ -4,18 +4,18 @@ import { jsx } from '@westpac/core';
 import PropTypes from 'prop-types';
 
 import { Button } from './Button';
-import { Select } from './Select';
-import { Label } from './Label';
+import { SelectField } from './SelectField';
+import { Text } from './Text';
```

Comment:
?

Reply:
`SelectField` renders our `<select>` and `<label>` add-on

`Text` is our text add-on

Question?